meshnets/utils/model_loading.py

```diff
@@ -0,0 +1,51 @@
+"""Methods for loading models"""
+
+import os
+import tempfile
+from typing import Union
+
+import mlflow
+# pylint: disable=unused-import
+from torch import tensor  # used to eval tensors
+
+from meshnets.modules.model import MeshGraphNet
+from meshnets.modules.lightning_wrapper import MGNLightningWrapper
+from meshnets.utils.datasets import FromDiskGeometricDataset
+
+
+def load_model_mgn(tracking_uri: str, run_id: str, ckpt: Union[int, str],
+                   dataset: FromDiskGeometricDataset) -> MGNLightningWrapper:
+    """Load a model from an MLFlow run and a given checkpoints.
+    
+    The checkpoint can be specified as either an index or the name of the
+    checkpoint."""
+
+    client = mlflow.tracking.MlflowClient(tracking_uri=tracking_uri)
+    run = client.get_run(run_id)
+
+    params = run.data.params
+    # pylint: disable=eval-used
+    params = {k: eval(v) for k, v in params.items()}
```

Comment:
Is there an alternative to using `eval`?

Reply:
Because the parameters are read as strings I am not sure, but this method is now deprecated